dns/octodns-config-validate.yaml

```diff
@@ -0,0 +1,72 @@
+---
+providers:
```

Comment:
Can you comment in here on how this file is different from the real file (and comment in both files that they need to be kept in sync) ?